code/modules/projectiles/boxes_magazines/ammo_boxes.dm

```diff
@@ -75,6 +75,9 @@
 	ammo_type = /obj/item/ammo_casing/c45
 	max_ammo = 20
 
+obj/item/ammo_box/c45/m45 // 1911 version with 3 loads
+	max_ammo = 24
+
```

Comment:
This should be moved to a file in the nsv13 folder

Reply:
fixed with [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)